go.mod

```diff
@@ -232,6 +232,8 @@ require (
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
 
+replace github.com/alecthomas/jsonschema => github.com/sguiheux/jsonschema v0.1.1
```

Comment:
pin to the right commit on the original repo

Reply:
done